articles/storage/storage-python-how-to-use-table-storage.md

```diff
@@ -102,13 +102,12 @@ use the **TableBatch** class. When you do want to submit the
 batch, you call **commit\_batch**. Note that all entities must be in the same partition in order to be changed as a batch. The example below adds two entities together in a batch.
 
 ```python
-from azure.storage.table import TableBatch
-batch = TableBatch()
+batch = table_service.begin_batch()
```

Comment:
@asears Hi Andrew, thanks for your contribution! When executing this line, the following error is returned:

`AttributeError: 'TableService' object has no attribute 'begin_batch'`

A quick look at [tableservice.py](https://github.com/Azure/azure-storage-python/blob/master/azure/storage/table/tableservice.py) and the [documentation for TableService](http://azure-storage.readthedocs.io/en/latest/ref/azure.storage.table.tableservice.html?highlight=tableservice) also shows no `begin_batch` in the TableService module.

Can you take another look at this code snippet and ensure the code functions, or perhaps give a few more details on how you have it working as it is currently?

Thanks!